### ISSUE TITLE
Align parameters of class comments

### ISSUE THREAD

**Issue by mirucon** [squizlabs/PHP_CodeSniffer#2379]
Say I have something like this:

```php
/**
 * @property string $email
 * @property string|null $name
 * @property string $password
 */
class User 
```

I would like to align these parameter comments just like what we can do for function:

```
/**
 * @property string      $email
 * @property string|null $name
 * @property string      $password
 */
class User 
```

Is there any way to achieve this? Seems like `PEAR.Commenting.ClassComment` can't do it for now.

**Comment by mirucon** [comment]
These comments are automatically generated by https://github.com/barryvdh/laravel-ide-helper and it's the reason why I use `@property` tag and can't go with class properties. If I submit a PR for this, would you be accepting or should I create an own sniff and distribute as another package?

**Comment by gsherwood** [comment]
I'd be happy to include a sniff into the Generic standard that looked for, and aligned, @property tags in class comments. I just can't accept a change to the PEAR sniff as it's not part of the PEAR standard.

**Comment by mirucon** [comment]
Thanks! I'll try this on weekends :)

**Comment by vladyslavstartsev** [comment]
@mirucon see also the https://github.com/slevomat/coding-standard may be they have something for this one
